Update travis file with docker py3qt5

mrosanes opened the pull request:
Update travis file to allow tests on docker taurus-test
with python3 and qt5.
taurus-test:debian-stretch-py3qt5

cpascual commented on lib/taurus/core/tango/tangoattribute.py:
```diff
@@ -328,7 +328,12 @@ def isFloat(self, inc_array=False):
+        # while taurus still compatible with these versions.
+        # PyTango.is_bool_type(tgtype, inc_array=inc_array)
+        return tgtype in (PyTango.CmdArgType.DevVarBooleanArray,
+                          PyTango.CmdArgType.DevBoolean)
```
This solution ignores the `inc_array` arg.

A better one would be:
```python
if tg_type == PyTango.CmdArgType.DevBoolean:
    return True
if inc_array and tg_type == PyTango.CmdArgType.DevVarBooleanArray:
    return True
return False
```

cpascual commented:
Hi, I manually made a small fix and merged (manually) as 300b409. So I close this PR

Commit:
Update travis file with docker py3qt5

Update travis file to allow tests on docker taurus-test
with python3 and qt5.
taurus-test:debian-stretch-py3qt5
